pandas/tests/io/test_excel.py

```diff
@@ -2182,6 +2182,8 @@ def test_write_cells_merge_styled(self, merge_cells, ext, engine):
             assert xcell_b1.font == openpyxl_sty_merged
             assert xcell_a2.font == openpyxl_sty_merged
 
+    @pytest.mark.xfail(not PY36, reason='only fails on Linux?',
```

Comment:
What failure are you getting here exactly?

Reply:
@WillAyd 
Those come from #24942, see e.g. the CI failures for one of the commits where I specifically reverted those xfails:
https://travis-ci.org/pandas-dev/pandas/builds/506535598?utm_source=github_status&utm_medium=notification
https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=9457